statistics/table.go

```diff
@@ -382,7 +382,11 @@ func (t *Table) ColumnBetweenRowCount(sc *stmtctx.StatementContext, a, b types.D
 	if !ok || c.IsInvalid(sc, t.Pseudo) {
 		return float64(t.Count) / pseudoBetweenRate
 	}
-	return c.betweenRowCount(a, b) * c.getIncreaseFactor(t.Count)
+	count := c.betweenRowCount(a, b)
+	if a.IsNull() {
```

Comment:
It's better to move this judgement into `Histogram.betweenRowCount`?

Reply:
Hmm, `Histogram` does not know it is a column histogram or index histogram, so I add this null accounting logic in the layer of `Column` and `Index`, I think this should be clearer?